### PR TITLE
feat(auto-updates): add support to notify users for an update on linux/msi COMPASS-7686

### PR DESCRIPTION
**Pull request by mabaasit**
### Checklist
- [ ] New tests and/or benchmarks are included
- [ ] Documentation is changed or added
- [ ] I have signed the MongoDB Contributor License Agreement (https://www.mongodb.com/legal/contributor-agreement)

## Motivation and Context

- [ ] Bugfix
- [ ] New feature
- [ ] Dependency update
- [ ] Misc

## Types of changes

- [ ] *Backport Needed*
- [ ] Patch (non-breaking change which fixes an issue)
- [ ] Minor (non-breaking change which adds functionality)
- [ ] Major (fix or feature that would cause existing functionality to change)

### CODE REVIEW

File: packages/compass/src/main/auto-update-manager.ts [hunk 1]
```diff
@@ -461,6 +500,55 @@ const STATE_UPDATE: Record<
...
+          newVersion: updateInfo.to,
+        });
+        return;
+      }
+
+      const answer = await dialog.showMessageBox({
+        icon: COMPASS_ICON,
+        title: 'New version available',
+        message: 'A new version of Compass is available to install',
+        detail: `Compass ${updateInfo.to} is available. You are currently using ${updateInfo.from}. Would you like to download it now?`,
```
Comment (addaleax) [inline packages/compass/src/main/auto-update-manager.ts:533]: Can we use the same text here for the new toast as well? It reads a bit nicer tbh :)

**Review by addaleax — APPROVED** [review]

File: packages/compass/src/main/auto-update-manager.ts (hunk 1 above)
Comment (mabaasit) [reply packages/compass/src/main/auto-update-manager.ts:533]: I reached out to Misba regarding the toast content and I'd share this idea (to have same for both) with her as well.

**Review by lerouxb — APPROVED** [review]
Looks good. We should probably manually test that this works, though. How would we do that? Multiple releases?

**Comment by mabaasit** [comment]
> Looks good. We should probably manually test that this works, though. How would we do that? Multiple releases?

Yes, at least 2 betas - first with these changes and then the next one should notify user of updates.

**Pull request merged**